reactComponents/dist/index.d.ts

```diff
@@ -1,5 +1,18 @@
+import { FC } from 'react';
 export { Accordion, AccordionItem, AccordionItemButton, AccordionItemHeading, AccordionItemPanel } from 'react-accessible-accordion';
 
-declare function SearchBar(): JSX.Element;
+declare type RenderProps = {
+    customWrapper?: object;
```

Comment:
we should type these instead of having the readme telling us what can be set

Reply:
customWrapper?:{
backgroundColor: string,      
    borderRadius: string,    
    border: string,
    button: {
        borderColor: string,
        backColor: string,
        color: string,
        borderRadius: string,
        fontSize: string,
        padding: string,
        hoverBackColor: string
    }
}

But make it so that each of these can be undefined as well

Reply:
The type has been added

Reply:
i still see the object type here - looks to me this is where the types matter as it refines the props

Reply:
I defined the types for the props correctly in types.tsx file